schema-loader/README.md

```diff
@@ -8,8 +8,8 @@ There are two ways to specify general CLI options in Schema Loader:
   - Pass a Scalar DB configuration file and database/storage-specific options additionally.
   - Pass the options without a Scalar DB configuration (Deprecated).
 
-Note that this tool supports only basic options to create/delete/repair a table.
-If you want to use advanced features of the database, please alter your tables after creating them with this tool.
+Note that this tool supports only basic options to create/delete/repair/alter a table. If you want
+to use advanced features of the database, please alter your tables after creating them with this tool.
```

Comment:
```suggestion
to use the advanced features of a database, please alter your tables with a database-specific tool after creating them with this tool.
```
Not fully sure, but this is probaly what you mean?

Reply:
Indeed, thank you. Good catch.